### PR TITLE
Publish TrueCommand 2.1 documentation

### PR DESCRIPTION
**Pull request by Mrt134**
Single PR to collect TrueCommand 2.1 documentation effort into one branch for faster publishing on release day.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/ReleaseNotes/TrueCommand/2.1.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+* **TrueNAS SCALE**: Manage a group of systems running a TrueNAS SCALE cluster. 
+
+## 2.1 Changelog
+
+### New Feature
+
+<ul>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1184'>TC-1184</a>] -         Add two-factor authentication support
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1581'>TC-1581</a>] -         Have TC auto-generate and use an auth token after initial NAS connect
```
Comment (micjohnson777) [inline content/ReleaseNotes/TrueCommand/2.1.md:23]: Maybe use  Has TC auto-generate and can use an auth token after intial NAS

File: content/ReleaseNotes/TrueCommand/2.1.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+### Improvement
+
+<ul>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1468'>TC-1468</a>] -         EULA needs to identify GPL components
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1489'>TC-1489</a>] -         Question about the Dashboard System Options Menu
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1603'>TC-1603</a>] -         Update selenium tests
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1655'>TC-1655</a>] -         Include "Group" or "All" option for system selection for reports
```
Comment (micjohnson777) [inline content/ReleaseNotes/TrueCommand/2.1.md:44]: Best practices is to Never use " " for emphasis, if a UI element use bold if exactly as it appears on-screen change Group and All to bold.

File: content/ReleaseNotes/TrueCommand/2.1.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1663'>TC-1663</a>] -         Add email verification to user email
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1772'>TC-1772</a>] -         Multiple time formats in use
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1789'>TC-1789</a>] -         Alerts for failed/suspicious login activity on a NAS
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1806'>TC-1806</a>] -         Remove PostgreSQL and migration routine.
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1811'>TC-1811</a>] -         Add "Test" button for LDAP settings
```
Comment (micjohnson777) [inline content/ReleaseNotes/TrueCommand/2.1.md:54]: Remove "" and use Bold for **Test**

File: content/ReleaseNotes/TrueCommand/2.1.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1821'>TC-1821</a>] -         Remove ng2-validation dependency
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1834'>TC-1834</a>] -         Add verbose logging and log level config
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1835'>TC-1835</a>] -         Update SMR disk model scanning
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1844'>TC-1844</a>] -         Prune dead code
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1850'>TC-1850</a>] -         Add "warning" to Cluster feature
```
Comment (micjohnson777) [inline content/ReleaseNotes/TrueCommand/2.1.md:70]: Remove " " around warning.

File: content/TrueCommand/TCGettingStarted/UserAccounts.md [hunk 5]
```diff
@@ -8,50 +8,71 @@ weight: 30
...
 
+## Two Factor Authentication
+
+
+Two-factor authentication is used to double check the authentication of an account user.
+The first verification occurs when the user logs in with a username and a password.
+Two-factor authentication adds an extra step in the process, a second security layer, that re-confirms their identity.
+If basic password security measures are in place, two-factor authentication makes it more difficult for unverified users to log in to your account.
+
+Enabling Two Factor Authentication requires an email address that has been authenticated. Authenticating a user's email address requires first setting up [SMTP Email]({{<relref "/TrueCommand/alerts/alertmanage.md" >}}) in **Settings -> Alert Services**.
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/UserAccounts.md:42]: for consistency do we want to say two-factor authentication or change line 37 to two factor authentication? Also, factor and authentication don't need to be init-capped.

Comment (Mrt134) [reply content/TrueCommand/TCGettingStarted/UserAccounts.md:42]: two-factor authentication is what I see typically used elsewhere - we should be consistent on that usage.

File: content/TrueCommand/TCGettingStarted/ConnectingTrueNAS.md [hunk 6]
```diff
@@ -1,31 +1,35 @@
...
+| Setting | Description |
+|------|------|
+| IP Address or Hostname | The system's IP address or DNS hostname. |
+| Nickname | Required short-form identifier for this system. You cannot use system nicknames more than once. |
+| Password / API Key | New password or API key. TrueCommand hades characters for security. |
+| Password / API Key Confirm | Re-enter the password or API key. |
 
-![Systems List](/images/TrueCommand/2.0/SystemsPage.png "Systems List")
+Enter the system IP address or DNS hostname, type a system nickname, and the system password.
+If you make a mistake, click **RESET FORM** to clear all entries and start over. .
```
Comment (micjohnson777) [inline content/TrueCommand/TCGettingStarted/ConnectingTrueNAS.md:24]: line 22 has extra spaces then a second period at the end of the line.

**Review by tonyriv3 — APPROVED** [review]

File: content/TrueCommand/SystemManagement/NASUsersAndGroups.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+If you want the user to have a home directory, enter the path to the directory and set the default UNIX permissions, then click **NEXT**.
+{{< /tab >}}
+
+{{< tab "Authentication" >}}
+![NASUser4](/images/TrueCommand/2.0/NASUser4.png "Creating a New NAS User")
+
+Configure the user's authorization settings.
+
+You can enter or paste the user's public SSH key in the *Sshpubkey* field.
```
Comment (micjohnson777) [inline content/TrueCommand/SystemManagement/NASUsersAndGroups.md:52]: change *Sshpubkey* to **Sshpubkey** if a UI element (above says it is the field name).

**Comment by Mrt134** [comment]
pgp.mit.edu is timing out again (slow website). Approved for merge.

**Review by micjohnson777 — COMMENTED** [review]
only minor things found that don't need to be changed for this release. let's rock n roll

**Pull request merged**